backend/Squirrel.SqlService/Squirrel.SqlService.BLL/Models/DTO/DbStructureDto.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using Squirrel.SqlService.BLL.Models.DTO.Function;
 using Squirrel.SqlService.BLL.Models.DTO.Procedure;
+using Squirrel.SqlService.BLL.Models.DTO.View;
 
 namespace Squirrel.SqlService.BLL.Models.DTO;
 
 public class DbStructureDto
 {
     public List<TableStructureDto>? DbTableStructures { get; set; } = new();
-    public TableConstraintsDto? DbConstraints { get; set; } = new();
+    public List<TableConstraintsDto>? DbConstraints { get; set; } = new();
```

Comment:
List of Lists? 

`TableConstraintsDto` already contains `public List<Constraint> Constraints { get; set; } = new();`

Reply:
Yes, but it's constraints for only one table